aiohttp/__init__.py

```diff
@@ -25,6 +25,7 @@
     ContentTypeError,
     Fingerprint,
     InvalidURL,
+    InvalidUrlRedirectClientError,
```

Comment:
Why expose this one specifically but not the others?

Reply:
added

Reply:
added all